libs/application/templates/estate/src/forms/OverviewSections/assetsAndDebts.ts

```diff
@@ -51,11 +51,7 @@ export const overviewAssetsAndDebts = [
               (asset.marketValue ? formatCurrency(asset.marketValue) : '0 kr.'),
             m.propertyShare.defaultMessage +
               ': ' +
-              (asset.share
-                ? asset.share > 1
-                  ? asset.share + '%'
-                  : asset.share * 100 + '%'
-                : '0%'),
+              (asset.share ? asset.share + '%' : ''),
```

Comment:
some of these might cause conflicts with https://github.com/island-is/island.is/pull/13055

Reply:
The mappers have been rewritten to give the original value in percentages.
Is there something I'm missing here?

The value of share will now always be as a percentage, no [0,1] ratios